src/main/resources/messages.properties

```diff
@@ -829,6 +830,12 @@ cannot.attack.civilians=&cYou cannot attack civilians!
 cannot.attack.clan.member=&cYou cannot attack another clan member!
 cannot.attack.ally=&cYou cannot attack an ally!
 cannot.send.discord.message=You cannot send messages to {0}. You''re not a member of that clan!
+clan.doesnt.have.any.linked.player=Clan {0} doesn't have any linked players
```

Comment:
Not necessary to mention the clan name/tag for the player. They already know it.

(Same for the other messages)

Reply:
> Not necessary to mention the clan name/tag for the player. They already know it.
Debug don't know it. 🍵

Reply:
Separate messages then.